test/commands/multi.spec.js

```diff
@@ -64,9 +64,9 @@ describe("The 'multi' method", function () {
                     // Provoke an error at queue time
                     multi1 = client.MULTI();
                     multi1.mset("multifoo", "10", "multibar", "20", helper.isString("OK"));
-                    multi1.set("foo2", helper.isError());
-                    multi1.incr("multifoo", helper.isNumber(11));
-                    multi1.incr("multibar", helper.isNumber(21));
+                    multi1.set("foo2");
+                    multi1.incr("multifoo");
+                    multi1.incr("multibar");
```

Comment:
Hm, if I look at the builds that are installed it says it's redis 2.8.21? ^^ If this is indeed an issue with windows, I'd like to wrap the helper.checks into another helper function to only include the check if it's not windows. That way we're still able to test everything on linux